setup.py

```diff
@@ -148,7 +148,7 @@ def build_extensions(self):
             # (-fPIC), since we use the python compiler flags
 
             link_objects = glob.glob(
-                os.path.join(self.cfitsio_build_dir, '*.a'))
+                os.path.join(self.cfitsio_build_dir, '*.o'))
```

Comment:
This is a big change. It is needed to hide the symbols. We now link against the .o files as if we were bundle the C code directly, not just bundling a static lib.